dev-libs/ancient/ancient-2.0.0.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A collection of old decompression routines"
+DESCRIPTION="This is a collection of decompression routines for various old formats that \were populare on Amiga, Atari computers and other old systems. A simple \command-line application is provided aswell."
+HOMEPAGE="https://github.com/temisu/ancient"
+COPYRIGHT="2017-20222, Teemu Suutari and contributors"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/temisu/ancient/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="9c6b31e4d968ec3adb25596a0b30405d1080d3de0546f1485ecb0ac2eca6261d"
+SOURCE_DIR="ancient-$portVersion"
+
+ARCHITECTURES="all ?x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="2.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	ancient$secondaryArchSuffix = $portVersion
+	cmd:ancient = $portVersion
```

Comment:
This should be `cmd:ancient$secondaryArchSuffix` or the recipe should be set up for `$commandBinDir` style of `cmd` entries

Reply:
I used $secondaryArchSuffix for now, unsure if $commandBinDir is needed. I do not know how to test

Reply:
(it went missing in my commit, it will be updated very soon - just doing a build test first)

Reply:
While this is fine for primary architecture on 64bit where `$binDir` is the same as `$prefix/bin`, on 32bit you need to use secondary architecture where `$binDir` is the same as `$prefix/bin/x86`, so best to use $commandBinDir here, if you need guidance just give a yell :)